### ISSUE TITLE
Label value and score do not match for Token.get_tag() and Token.get_tags_proba_dist() 

### ISSUE THREAD

**Issue by will-henderson** [zalandoresearch/flair#877]
`tagger = SequenceTagger.load('ner-ontonotes')

sentence = Sentence('( 1 ) The law is based on the EU EAPO Regulation ( 655 / 2014 ) .')
sentence = tagger.predict(sentence)

token = sentence[0].tokens[1]
tags = [tag for tag in token.get_tags_proba_dist('ner') if tag.score > 0.01]
print(tags)

tags = token.get_tag('ner')
print(tags)
`

This prints:

[O (0.35663124918937683), B-CARDINAL (0.22679153084754944), S-CARDINAL (0.37637460231781006), B-PERCENT (0.024165131151676178)]

O (0.37637460231781006)

In these two cases, O has a different score, and in the second case, O's score is that of S-Cardinal in the first case. I have looked at the SequenceTagger class to try to understand this, but  I haven't been able to. Is it to do with pytorch's softmax function? 

Thanks

**Comment by alanakbik** [comment]
Hi @will-henderson this is probably the same as the error that's being fixed with PR #949!

**Comment by alanakbik** [comment]
I believe this is fixed in #949, i.e. it should be fixed in the master branch. Should the error persist please reopen and let us know.
